build/Version.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <VersionMajor>2</VersionMajor>
     <VersionMinor>1</VersionMinor>
-    <VersionPatch>300</VersionPatch>
+    <VersionPatch>400</VersionPatch>
```

Comment:
This flows to the installer branding now, right?

Reply:
Yes, but I forgot to update the suffix to preview. Will push a better commit.